eo-runtime/src/test/java/EOorg/EOeolang/EODataizeAppCallsVarargsFuncTest.java

```diff
@@ -0,0 +1,125 @@
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package EOorg.EOeolang;
+
+import org.eolang.AtComposite;
+import org.eolang.AtOnce;
+import org.eolang.AtVararg;
+import org.eolang.Data;
+import org.eolang.Dataized;
+import org.eolang.PhCopy;
+import org.eolang.PhDefault;
+import org.eolang.PhMethod;
+import org.eolang.PhWith;
+import org.eolang.Phi;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests dataization of an EO app that calls a function
+ * with varargs.
+ *
+ * @since 0.22
+ * @todo #414:30min Fix bug EO app calling a varargs func.
+ *  We reproduced by a Java and EO tests, the bug with exception `You can't overwrite X`
+ *  when EO app try to call a function that uses varargs as parameter. Now, we must fix it
+ *  and enable test below and eo test {@code [] > calls-varargs-func} located in
+ *  `runtime-tests.eo`.
+ */
+final class EODataizeAppCallsVarargsFuncTest {
+
+    @Test
+    @Disabled
+    public void dataizesEOappThatCallsVarargsFunc() {
+        MatcherAssert.assertThat(
+            new Dataized(
+                new EOappThatCallsVarargsFunc(Phi.Φ)
+            ).take(Long.class),
+            Matchers.is(2L)
+        );
+    }
+
+    /**
+     * Main program sample.
+     * {@code
+     *  [] > app
+     *    (f 1 2 3).eq 2 > @
+     * }
+     * @since 0.22
+     */
+    private static class EOappThatCallsVarargsFunc extends PhDefault {
+        public EOappThatCallsVarargsFunc(Phi sigma) {
+            super(sigma);
+            this.add(
+                "φ",
+                new AtOnce(
+                    new AtComposite(
+                        this,
+                        (rho) -> {
+                            Phi fvar = new PhCopy(new Fvarargs(rho));
+                            fvar = new PhWith(fvar, 0, new Data.ToPhi(1L));
+                            fvar = new PhWith(fvar, 1, new Data.ToPhi(2L));
+                            fvar = new PhWith(fvar, 2, new Data.ToPhi(3L));
+                            return new PhWith(
+                                new PhCopy(new PhMethod(fvar, "eq")), 0, new Data.ToPhi(2L)
+                            );
+                        }
+                    )
+                )
+            );
+        }
+    }
+
+    /**
+     * Varargs function sample.
+     * {@code
+     *  [args...] > f
+     *    1 > a
+     *    2 > @
+     * }
+     * @since 0.22
+     */
+    @SuppressWarnings("unchecked")
+    private static class Fvarargs extends PhDefault {
+        public Fvarargs(final Phi sigma) {
+            super(sigma);
+            this.add("args", new AtVararg());
+            this.add(
```

Comment:
@baudoliver7 why do you need this attribute here? It doesn't seem to be relevant to tests... what it's for?

Reply:
@yegor256 it's to reproduce the function :
```eo
[args...] > f
  1 > a
  2 > @
```
The error appears only when there is another attribute. It vanishes when we write :
```eo
[args...] > f
  2 > @
```